### DISCUSSION TITLE
Knowledge Agents Intrinsics

### DISCUSSION THREAD

**Discussion by drcgjung** [eclipse-tractusx/sig-release#254] [Ideas]
This is a placeholder for the topics which surfaced in the Agents Kits for release 24.03 without any dependency

**Comment by igorsvetlov** [comment]
### Story "Move postman, docker-compose, resources and sample umbrella chart to eclipse-tractusx/tutorials-ressources"

**Description:**
As an eclipse commiter, I want to get a PR on the new eclipse-tractusx/tutorial-ressources repository which contains the sample dataspace (local docker-compose or cluster-based umbrella chart) of Knowledge Agents including needed ressources.

**AC:**
- Umbrella chart should use the eclipse-tractusx chart instead of custom stuff.
- An initial alignment with other test dataspaces has been made.
- Secrets are replaced by a common/shared mechanism that has been aligned
- use predefined virtualization engines if possible.

**Estimated Story Points:**
3SP

**Comment by igorsvetlov** [comment]
### Story "Describe and Evaluate Concepts for Enforcing Linked Nodes between Business Partners"

**Description:**
As a dataspace consumer, I want the business partners to standardize the instance nodes IRIs such that I can easily and consistently link data between different assets.

**AC:**
- Several concepts for IRI creation
- Evaluation wrt data sovereignity and versioning

**Estimated Story Points:**
4SP

**Comment by igorsvetlov** [comment]
### Story "Synchronize the federated data catalogue by a predefined skill"

**Description:**
As an agent-enabled dataspace participant, I want to be able to dynamically influence/determine the catalogues with which my connector synchronized such that I do not have to reconfigure my agent plane regularly.

**AC:**
- Instead of the fixed list of connectors, the synchronizer executes a skill to get the list of connectors to synchronize with
- skill can be configured in EDC properties/environment
- skill can be configured in HELM chart

**Estimated Story Points:**
5SP

**Comment by igorsvetlov** [comment]
### Story "Define the list of partner BPNLs via configuration or a skill execution"

**Description:**
As an agent-enabled dataspace participant, I want to have an easy/dynamic method to define the agent connector environment such that I do not have to reconfigure my agent plane regularly.

**AC:**
- no more need to setup initial federated data catalogue
- configuration option in Agent Plane
- configuration option in Helm Chart
- BPNLs as a list
- BPNLs as the result of a skill execution
- Business partner entities and their EDC urls are automatically
- configuration of Discovery Service + Service Account in Agent Plane
- configuration of Discovery Service + Service Account in Helm Chart

**Estimated Story Points:**
10SP

**Comment by igorsvetlov** [comment]
### Story "Create Product Pass Ontology"

**Description:**
Create a generic ontology 'Product Pass' that can be specialized to individual products. Similar to 'Vehicle Information'.

**AC:**
TBD

**Estimated Story Points:**

**Comment by igorsvetlov** [comment]
### Story "Prepare and Perform a Demo"

**Description:**
As a KA stakeholder, I want to see a Demo of simple graph provisioning feature such that I can decide to deploy the final release.

**AC:**
-Demo script/postman
-Demo Environment
-Demo presentation
-Sample data

**Estimated Story Points:**
4SP

**Comment by stephanbcbauer** [comment]
@drcgjung since this discussion is ancient and my assumption is you used this for feature planning in older releases, I close this discussion.
